packages/wonder-blocks-switch/src/theme/index.ts

```diff
@@ -0,0 +1,8 @@
+import {mapValuesToCssVars} from "@khanacademy/wonder-blocks-tokens";
+import themeDefault from "./default";
+
+// Export the raw theme for direct use in components
+export const theme = themeDefault;
+
+// Export the CSS variables for use in CSS
+export default mapValuesToCssVars(themeDefault, "--wb-c-switch-");
```

Comment:
Do we need this file?

Reply:
See my other comment re `root.border.radius`. We would still need this.